code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1270,3 +1270,20 @@ datum/reagent/romerol
 	M.resize = 1/current_size
 	M.update_transform()
 	..()
+	
+/datum/reagent/hormone
+	name = "Exotic Hormone"
+	id = "hormone"
+	description = "Swaps your Gender"
+	reagent_state = LIQUID
+	color = "#78FFF0"
+	metabolization_rate = 5
+
+/datum/reagent/hormone/on_mob_delete(mob/m)
+	if(method == INJECT && iscarbon(m))
```

Comment:
Pretty sure this can never return `1`; `INJECT` is not stored in the reagent's data, as far as I know, and is only ever proc'd or checked for when the reagent is added to someone.